thoughts/shared/research/2026-02-22-unkey-api-layer-integration.md

```
@@ -0,0 +1,339 @@
+repository: lightfast-search-perf-improvements
+topic: "Implementing Unkey.com for the v1 API Layer"
+tags: [research, codebase, unkey, api-keys, auth, rate-limiting, v1-api]
+status: complete
+last_updated: 2026-02-22
+last_updated_by: claude-sonnet-4-6
+---
+
+# Research: Implementing Unkey.com for the v1 API Layer
+
+**Date**: 2026-02-22T05:32:51Z
+**Git Commit**: c510c99b
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)